### PR TITLE
Feat : Filter and Product List features

### PR DESCRIPTION
**Pull request by anuj-kumary**
- I have added the Filter section and Product list Section
Note:- It's not responsive I am going to work on responsiveness in the upcoming days.

Screenshot - 
Filter and Product

### CODE REVIEW

File: Pages/Product List/product-list.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    opacity: 0.7;
+}
+.filter__input{
+    padding: .3rem 0;
+}
+
+.product{
+   width: 80vw;
+   margin: 0 auto;
+   background-color: #f5f1f1;
```
Comment (rahulbakshi4) [inline Pages/Product List/product-list.css]: Use variables for colors if they are repeating

**Review by rahulbakshi4 — APPROVED** [review]
Add variables for colors rest looks good !!

File: Pages/Product List/product-list.css (hunk 1 above)
Comment (anuj-kumary) [reply Pages/Product List/product-list.css]: Done

File: Pages/Product List/product-list.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Listening</title>
```
Comment (Parul1999) [inline Pages/Product List/product-list.html]: Product Listing

File: Pages/Product List/product-list.html [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                    <button class="btn btn__primary">Add to Cart</button>
+                </div>
+              </div>
+
+              <div class="product__card">
+                <div class="product__image">
+                    <img src="https://www.apple.com/v/mac/home/bk/images/overview/compare/compare_mba__fchj615oz0yi_medium_2x.png" alt="MacBook Pro">
+                    <span class="product__favourite"><i class="far fa-heart"></i></span>
+                </div>
+                <div class="product__heading">
```
Comment (Parul1999) [inline Pages/Product List/product-list.html]: Semantic tags can be used

**Review by Parul1999 — APPROVED** [review]
Semantic tags can be used for headings. The rest looks good.

File: Pages/Product List/product-list.html (hunk 2 above)
Comment (anuj-kumary) [reply Pages/Product List/product-list.html]: Done

**Review by Shah-Arsalan — APPROVED** [review]

**Comment by ddivyansh18** [comment]
Looks good to merge now !

**Review by ddivyansh18 — APPROVED** [review]

**Pull request merged**